package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "resto",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "start": "npm run lite",
+    "lite": "lite-server"
+  },
```

Comment:
Great work adding the scripts. 👏🏿  However, let's improve on them. Kindly check below: 

Your lite server is causing problems when running the project locally. I was unable to run your project. let's improve on this by using a web pack instead. ⬇️ 

- [ ]  [required]  Kindly add the correct web pack scripts. This will allow the web pack to be correctly configured in your project and we shall be able to use its great features such as a web server.  You can do this by configuring the right scripts ⬇️ 

```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```